repository-data/site/src/main/resources/hcm-config/hst/hosts/prd-brcloud/uk.yaml

```diff
@@ -32,10 +32,8 @@ definitions:
               hst:homepage: root
               hst:locale: en
               hst:mountpoint: /hst:hee/hst:sites/dental
-              hst:responseheaders: ['robots: noindex, nofollow', 'Content-Security-Policy:
-                  frame-ancestors ''self''', 'X-Frame-Options: SAMEORIGIN']
-              hst:roles: [site.viewer]
-              hst:authenticated: true
+              hst:responseheaders: ['Content-Security-Policy: frame-ancestors ''self''',
```

Comment:
Could we get rid of `hst:responseheaders` as it would be inherited from `/hst:hee/hst:hosts/prd-brcloud/uk` ?